### ISSUE TITLE
Remove ext quick access

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#152371]
This menu is about to be much more visible, which is great, because most people don't know about these features, but we should make sure that all of them are still relevant.

`ext` and `ext install` just redirect to the extension viewlet and don't seem useful to promote

![image](https://user-images.githubusercontent.com/323878/174140803-3c4bafa2-cd1c-4963-b580-54f297aac563.png)

We should also be thinking about other useful ones that could be added.

cc @jrieken

**Comment by sandy081** [comment]
These were there from the time before we have extensions UI. I am fine removing them.

CC @joaomoreno

**Comment by joaomoreno** [comment]
I agree.

**Comment by bpasero** [comment]
I think market place was telling users to install an extension using quick pick until we added protocol handler. Not sure we still have mentions of this elsewhere.

**Comment by bpasero** [comment]
Btw I don't think this is from a time before we had UI, we just wanted to make the UI more discoverable and I would maybe first check with telemetry how often this is still being used before dropping it.

//cc @isidorn

**Comment by joaomoreno** [comment]
> I think market place was telling users to install an extension using quick pick until we added protocol handler. Not sure we still have mentions of this elsewhere.

Yup, I remembered the same and confirmed they are no longer doing that.

This is indeed from the time before we had UI but once we introduced UI, that pattern had spread to many places (Marketplace, docs) so we decided to keep it just so not to break those flows. But since I couldn't find it any longer in the Marketplace nor in our docs I think it's time to remove it.
